packages/node-cli/src/commands/AddDependenciesCommandGenerator.js

```diff
@@ -152,7 +161,8 @@ module.exports = class AddDependenciesCommandGenerator extends BaseCommandGenera
 		return dependenciesString;
 	}
 
-	_formatOutput(operationResult) {
+	_formatOutput(actionResult) {
+		const operationResult = actionResult._context.operationResult;
```

Comment:
isn't `_context` a private variable inside ActionResult? If yes, we should access with a getter, like `actionResult.getContext().operationResult`

Reply:
Yes, well context would not be part of actionResult anymore... I have developed only deploy and createproject.... add dependencies will be next in the list

Reply:
Added changes AddDependencies... please check. I leave this conversation opened until then